src/index.js

```diff
@@ -18,9 +56,13 @@ export async function getSubtitles({
 
   const decodedData = decodeURIComponent(data);
 
-  // * ensure we have access to captions data
-  if (!decodedData.includes('captionTracks'))
-    throw new Error(`Could not find captions for video: ${videoID}`);
```

Comment:
does this error need to continue existing if there's nothing alternative found?

Reply:
Well I check and throw from this alternative method, but it might be a good thing to check here as well to reduce coupling

Reply:
@bel0v how to avoid throwing error at all?